actors/power/src/state.rs

```diff
@@ -44,6 +44,8 @@ pub const CRON_QUEUE_AMT_BITWIDTH: u32 = 6;
 pub type ClaimsMap<BS> = Map2<BS, Address, Claim>;
 pub const CLAIMS_CONFIG: Config = DEFAULT_HAMT_CONFIG;
 
+const RAMP_DURATION_EPOCHS: u64 = 365 * 2880; // 1Y
```

Comment:
Let's make this a configurable parameter as well. Put it next to the ramp start epoch.